modules/persistent-storage-cinder-maximum-volume.adoc

```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent_storage-cinder.adoc
+:_content-type: CONCEPT
+[id="persistent-storage-cinder-maximum-volume_{context}"]
+= Maximum number of Cinder volumes on a node
+By default, {product-title} supports a maximum of 256 Cinder volumes attached to one node, and the Cinder predicate that limits attachable volumes is disabled. To enable the predicate, add `MaxCinderVolumeCount` string to the `predicates` field in the scheduler policy.
```

Comment:
Based on this, it isn't clear how to actually accomplish this? We don't actually describe where to define the `MaxCinderVolumeCount` field in the YAML. I can't find a reference to this in the API docs, either, so as a user I'm not sure how to accomplish this? What's a scheduler policy? Where do I find mine?

https://docs.openshift.com/container-platform/4.10/rest_api/workloads_apis/persistentvolume-v1.html#persistentvolume-v1

Reply:
I don't have a 4.8 cluster in-front of me, but basically the idea is to edit scheduler policy specified in `oc get scheduler cluster -o yaml` object. it has a `policy` field in the spec, which points to a configmap inside `openshift-config` namespace. 

I am from storage team, but someone from scheduler team might know better.

Reply:
`Additional Resources` section provides the link to know more about modifying the scheduler policy. If you follow the link it shows the `predicates` field where `MaxCinderVolumeCount` can be added.